README.md

```diff
@@ -27,12 +27,21 @@ To install the requirements first run the command
 ```
 sudo python3 setup.py install
 ```
+>Note: We only support Python 3.5
 
 >Note: there is no sudo for windows as well as when you have root privelages.
 Just run commands without sudo
 
 >Note: some setups use python instead of python3 and pip3 instead of pip
 
+### For Debian based Linux systems:
+
+Run these commands before running the above command:
+```
+sudo apt-get install -y python3-gi python3-gi-cairo libglib2.0-dev libffi-dev libgirepository1.0 libcairo2-dev python3-cairo-dev
```

Comment:
This holds up a lot, but I think it's better to build from source on the system, as specified [here](https://git.gnome.org/browse/pygobject/tree/README)